src/Jackett.Common/Definitions/xbytes2.yml

```diff
@@ -39,6 +39,8 @@
   search:
     path: index.php
     keywordsfilters:
+      - name: re_replace
+        args: ["S(\\d{1,2})E(\\d{1,2}) S(\\d{1,2})E(\\d{1,2})", "S$1/E$2"]
```

Comment:
This filter to workaround the sickrage bug is not acceptable. Please migrate to medusa or fix it either on the sickrage side or on the jackett side (for all indexers, not just one).